src/utils/config.py

```diff
@@ -6,3 +6,10 @@ class SweepConfig(BaseModel):
     include_exts: list[str] = ['.cs', '.csharp', '.py', '.md', '.txt', '.ts', '.tsx', '.js', '.jsx', '.mjs']
     exclude_exts: list[str] = ['.min.js', '.min.js.map', '.min.css', '.min.css.map']
     max_file_limit: int = 60_000
+
+    def get_exclusion_list(self):
+        try:
+            with open('exclusion.txt', 'r') as f:
```

Comment:
take this as an argument, this should come from repo get file where we look for a sweep.yaml